src/app/clusters/ota-provider/ota-provider.cpp

```diff
@@ -55,7 +55,7 @@ constexpr size_t kMaxMetadataLen       = 512; // The maximum length of Metadata
 constexpr size_t kUpdateTokenMaxLength = 32;  // The expected length of the Update Token parameter used in multiple commands
 constexpr size_t kUpdateTokenMinLength = 8;   // The expected length of the Update Token parameter used in multiple commands
 
-OTAProviderDelegate * gDelegateTable[kOtaProviderDelegateTableSize] = { nullptr };
+OTAProviderDelegate ** gDelegateTable = nullptr;
```

Comment:
Was this tested _at all_?

Reply:
I run the compilation with https://github.com/project-chip/connectedhomeip/pull/28316/files, which seems to be ok. It seems CI has tested the OTA.